### PR TITLE
Paycheck alteration at identification and departmental consoles

### PR DESCRIPTION
**Pull request by Dingo-Dongler**
## About The Pull Request

Adds a menu to ID consoles and department consoles to change the paychecks of crew, or gives them bonuses or debts. Chemist blows up their dispenser? Dock their pay! Botanist is doing great work? Give them a raise!

## Why It's Good For The Game

Adds an rp way to benefit or punish your subordinates. Heads need more boss-like powers.

## Changelog
:cl:
add:  You can change the pay of your employees at ID or head department consoles, or give them bonuses and debts. That's coming out of your paycheck!
/:cl:

### CODE REVIEW

File: code/game/machinery/computer/card.dm [hunk 1]
```diff
@@ -380,21 +424,21 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 						playsound(src, 'sound/machines/terminal_on.ogg', 50, 0)
 
 					else
-						if((ACCESS_HOP in scan.access) && ((target_dept==1) || !target_dept))
-							region_access |= 1
+						if((ACCESS_HOP in scan.access) && ((target_dept==DEPT_HOP) || !target_dept))
+							region_access |= DEPT_HOP
 							region_access |= 6
```
Comment (Dingo-Dongler) [inline code/game/machinery/computer/card.dm]: What is this |= 6 even for

**Comment by Dingo-Dongler** [comment]
suck my dick and balls travis

File: code/game/machinery/computer/card.dm [hunk 2]
```diff
@@ -528,6 +571,44 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 				to_chat(usr, "<span class='notice'>[j.title] has been successfully [priority ?  "prioritized" : "unprioritized"]. Potential employees will notice your request.</span>")
 				playsound(src, 'sound/machines/terminal_prompt_confirm.ogg', 50, 0)
 
+		if ("adjust_pay")
```
Comment (PowerfulBacon) [inline code/game/machinery/computer/card.dm:574]: You should add checks here to make sure the user has the ability to adjust pay and isn't just exploiting hrefs

File: code/game/machinery/computer/card.dm [hunk 3]
```diff
@@ -528,6 +571,44 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
...
+			if(isnull(new_pay))
+				updateUsrDialog()
+				return
+			if(new_pay < 0)
+				to_chat(usr, "<span class='warning'>Paychecks cannot be negative.</span>")
+				updateUsrDialog()
+				return
+			account.paycheck_amount = new_pay
+
+		if ("adjust_bonus")
```
Comment (ike709) [inline code/game/machinery/computer/card.dm:619]: ditto here

File: code/game/machinery/computer/card.dm (hunk 2 above)
Comment (Dingo-Dongler) [reply code/game/machinery/computer/card.dm:574]: So I'd have to recheck the departments that the user can adjust the pay of too?

**Review by PowerfulBacon — APPROVED** [review]
In before heads turn all of their workers into slaves

**Pull request merged**